src/components/navigation/categories.ts

```diff
@@ -36,6 +36,11 @@ export const navigationCategories = [
             url: '/data-availability',
             icon: 'ph-database ph-bold',
           },
+          {
+            title: 'Intents',
+            url: '/intents',
+            icon: 'ph-handshake ph-bold',
+          },
```

Comment:
For the navbar and the footer, I think this should go between `Chain Abstraction` and `Open Web Applications`.  That puts our most promoted parts of the stack (Blockchain, Abstraction, Intents) first and in bottom-up order.